src/applications/simple-forms/21P-0847/containers/ConfirmationPage.jsx

```diff
@@ -16,24 +16,24 @@ const childContent = (
       What are my next steps?
     </h2>
     <p>
-      Once a substitution request is granted, the substitute claimant must
-      complete any action required by law or regulation within the time period
-      remaining for the original claimant to take such action on the date of
-      their death. The substitute has the same rights as would have applied to
-      the original claimant had they not died, including the rights regarding
-      hearings, representation, legacy appeals, decision review options for a
-      claim not finally adjudicated, and submission of evidence. The substitue
-      also has the right to request a decision review of a deceased claimant’s
-      claim that is not yet final.
+      After we approve your substitution request you’re responsible for taking
```

Comment:
maybe a question for content - do you think we should have a comma after "substitution request"? not a blocking issue, just something to ask about.

Reply:
@jeana-adhoc ^

Reply:
Good eye @rhasselle-oddball. We missed transposing this from the word doc. Yes, it should read: 

`After we approve your substitution request, you're responsible...`

Reply:
Added the comma. Thanks!